locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -141,7 +144,10 @@ FROM node:boron
 WORKDIR /usr/src/app
 
 # Install app dependencies
-COPY package.json package-lock.json .
+COPY package.json .
+# For npm@5 or later, copy package-lock.json as well
```

Comment:
I did this on purpose, as it still belongs to the same block (and would replace line 147)